util/mp-functions.xsl

```diff
@@ -336,12 +336,14 @@
         <xd:param name="start-date">start date of the period</xd:param>
         <xd:param name="periode">duration of the period</xd:param>
         <xd:param name="end-date">end date of the period</xd:param>
+        <xd:param name="criterium">criterium of the period</xd:param>
     </xd:doc>
     <xsl:function name="nf:periode-string" as="xs:string?">
         <xsl:param name="current-bouwsteen"/>
         <xsl:param name="start-date"/>
         <xsl:param name="periode"/>
         <xsl:param name="end-date"/>
+        <xsl:param name="criterium" as="element()?"/>
```

Comment:
It would be nice to add the `as="element()?"` to the other params as well, since they appear to be optional (at least based on the if statements below). Or is there a reason for this difference between this param and the other ones?

Reply:
Agreed and done.